frontend/sozisel-app/src/components/Navbar/navbar.scss

```diff
@@ -0,0 +1,39 @@
+@use "../../index.scss";
+
+nav {
```

Comment:
If you create `scss` files for certain components name them the same as those components. In this case `Navbar.scss` and put all that tag styles under one `.Navbar` class and then apply `.Navbar` style to top container in `Navbar` component.